calico/about/product-comparison.mdx

```diff
@@ -0,0 +1,87 @@
+---
+description: Describes Tigera products, including a feature comparison table.
+---
+
+# Product comparison
+
+**{{prodname}} Open Source** is the base product that comprises both {{prodname}} Enterprise and {{prodname}} Cloud. It provides the core networking and network policy features.
+
+**{{prodname}} Enterprise** adds a web UI for visibility and troubleshooting Kubernetes clusters, and other value-added features on top of {{prodname}} Open Source. 
+
+**{{prodname}} Cloud** is the SaaS version of {{prodname}} Enterprise. It includes Image Assurance to scan and detect vulnerabilities in images, and container threat defense to detect malware. It also adds onboarding tutorials, and eliminates the cost to manage Elasticsearch logs and storage. 
+
+![tigera-product-overview](/img/calico/calico-products.svg)
+
+What is the best fit for you? It depends on your needs. The following table provides a high-level comparison.  
+
+| Product                   | Cost and support                                             | Best fit                                                     |
+| ------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| {{prodname}}  Open Source | Free, community-supported                                    | **Users** who need the flexibility and scalability of {{prodname}}, who want the more robust network policy but seamless support for Kubernetes policy, BGP networking, and the choice of dataplanes without any costs. |
```

Comment:
" who want the more robust network policy but seamless support for Kubernetes policy"

Feel like this phrasing is a little bit confusing. Perhaps just "Users who want best-in-class networking and network policy capabilities for Kubernetes without any costs" or something like that?

Reply:
Reworded.